src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+    "\n",
+    "Currently, IDAES supports calculation of capital costing for a wide array of unit operations:\n",
+    "\n",
+    "- Pressure Changers (Compressor, Turbine, Pump, generic PressureChanger)\n",
+    "- Temperature Changers (Heater, HeatExchanger, HeatExchangerNTU)\n",
+    "- Vertical and Horizontal Vessels (Flash, CSTR, PFR, StoichiometricReactor)\n",
+    "\n",
+    "The framework includes factors for vessel shell thickness and dimensions, material properties for equipment shells and tubes, and tray types for distillation columns. The framework does not yet support heat exchangers requiring both shell and tube area (HX1D).\n",
+    "\n",
+    "For capital costs calculations, IDAES supports the following options:\n",
+    "- Heat exchanger type:\n",
+    "    - floating head, fixed head, \"U tube\", Kettle evaporator\n",
+    "- Heat exchanger material:\n",
+    "    - pure Carbon Steel, Stainless Steel, Cr Mo Steel, Monel or Titanium; Carbon Steel alloy with Brass, Stainless Steel, Monel, Titanium or Cr Mo Steel; \n",
+    "- Heat exchanger tube lengths:\n",
+    "    - 8, 12, 16 or 20 feet\n",
+    "- Vessel materials\n",
+    "    - Carbon Steel, Low Alloy Steel, Stainless Steel 304, Stainless Stell 316, Carpenter 20CB3, Nickel 200, Money 400, Inconel 600, Incoloy 825, Titanium\n",
+    "- Column tray type:\n",
+    "    - Sieve, Valve, Bubble Cap\n",
+    "- Column tray material:\n",
+    "    - Carbon Steel, Stainless Stell 303, Stainless Stell 316, Carpenter 20CB3, Monel\n",
+    "- Heater material:\n",
+    "    - Carbon Steel, Cr Mo Steel, Stainless Steel\n",
+    "- Heater source:\n",
+    "    - Fuel, Reformer, Pyrolysis, Hot Water, Salts, Dowtherm A, Steam Boiler\n",
+    "- Compressor type:\n",
+    "    - Centrifugal, Reciprocating, Screw\n",
+    "- Compressor drive type:\n",
+    "    - Electric Motor, Steam Turbine, Gas Turbine\n",
+    "- Compressor material:\n",
+    "    - Carbon Steel, Stainless Steel, Nickel Alloy\n",
+    "- Pump Material:\n",
+    "    - Cast Iron, Ductile Iron, Cast Steel, Bronze, Stainless Steel, Hastelloy C, Monel, Nickel, Titanium, Ni Al Bronze, Carbon Steel\n",
+    "- Pump type:\n",
+    "    - Centrigual, External Gear, Reciprocating\n",
+    "- Pump motor type:\n",
+    "    - Open, Enclosed, Explosion Proof\n",
+    "- Fan type:\n",
+    "    - Centrifugal Backward, Centrifugal Straight, Vane Axial, Tube Axial\n",
+    "- Fan material:\n",
+    "    - Carbon Steel, Fiberglass, Stainless Steel, Nickel Alloy\n",
+    "- Blower type:\n",
+    "    - Centrifugal, Rotary\n",
+    "- Blower material:\n",
+    "    - Carbon Steel, Aluminum, Fiberglass, Stainless Steel, Nickel Alloy\n",
+    "\n",
+    "\n",
+    "The framework contains the following flowsheet costing methods and default options under the \"SSLWCosting\" class:\n",
```

Comment:
Again, refer to the docs for the comprehensive details. Here you need to take the user through the steps. Refer to the Diataxis docs for some guidance on what should go in a tutorial: https://diataxis.fr/tutorials/